rskj-core/src/test/java/co/rsk/test/dsl/WorldDslProcessor.java

```diff
@@ -121,14 +122,35 @@ private void processAccountNewCommand(DslCommand cmd) {
         String name = cmd.getArgument(0);
         builder.name(name);
 
-        if (cmd.getArity() > 1)
+        if (cmd.getArity() > 1) {
             builder.balance(new Coin(new BigInteger(cmd.getArgument(1))));
+        }
+
+        if (cmd.getArity() > 2) {
+            builder.code(Hex.decode(expandAccounts(cmd.getArgument(2))));
+        }
 
         Account account = builder.build();
 
         world.saveAccount(name, account);
     }
 
+    private String expandAccounts(String bytecodes) {
```

Comment:
what does this do? are you using this to simulate something related to create2? I don't really follow it.

Reply:
It's related with `delegatecall01.txt` test file:

```
account_new acc1 10000000

# contract account with simple return

# code
# PUSH1 0x00
# PUSH1 0x00
# RETURN

account_new delegated 0 60006000f3

# contract account with delegate call

# code
# PUSH1 0x00    (output data size)
# PUSH1 0x00    (output data offset)
# PUSH1 0x00    (input data size)
# PUSH1 0x00    (input data offset)
# PUSH20 <delegated contract address>
# PUSH2 0x1000   (gas to use)
# DELEGATECALL
# STOP

account_new delegatecall 0 600060006000600073[delegated]611000f400
```

it replace `[delegated]` by the address of that account. We needed that address in the code to invoke the call to that address. The bytecode `73  push the next 20 bytes into the stack.

There is also a test code `WorldDslProcessorTest.processAccountNewCommandWithBalanceAndCodeWithOtherAccountAddress`